src/components/Table/index.d.ts

```diff
@@ -11,6 +11,8 @@ export interface TableProps extends BaseProps {
     minColumnWidth?: string | number;
     maxColumnWidth?: string | number;
     showCheckboxColumn?: boolean;
+    showRowNumberColumn: boolean;
+    rowNumberOffset: number;
```

Comment:
remember no props are required, this is very important since in this way typescript user will must pass showRowNumberColumn and rowNumberOffset and then the current usages of the table would break, instead:
```
showRowNumberColumn?: boolean;
rowNumberOffset?: number;
```

Reply:
sure